babyProject/babyProject/Views/ViewController.swift

```diff
@@ -25,19 +30,20 @@ class ViewController: UIViewController {
         }
     }
     var result: bpRootResponse?
-    let rootURL = "https://api.turkuforge.fi"
 
     override func viewDidLoad() {
         super.viewDidLoad()
         preventLargeTitleCollapsing()
 
-        let url = URL(string: rootURL)!
+        let url = URL(string: GlobalValues.rootUrl)!
         self.cancellable = URLSession.shared.dataTaskPublisher(for: url)
             .map { $0.data }
             .decode(type: bpRootResponse.self, decoder: JSONDecoder())
             .eraseToAnyPublisher()
             .sink(receiveCompletion: { print ("Received completion: \($0).") },
-                  receiveValue: { response in self.channels = response.embedded.channelList
+                  receiveValue: { response in
+                    self.channels = response.embedded.channelList
+                    GlobalValues.sockjsEndpoint = response.links.sockJsEndpoint?.href
                     print("here is self channel first ______ \(response)")
```

Comment:
Do you need this print statement in production?

Reply:
I forgot to remove them :D